### PR TITLE
Feat/hub enterprise links

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/manager-hub`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? |no
| Tickets          | none
| License          | BSD 3-Clause

## Description

Update links for enterprise customers

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/user/user.service.js [hunk 1]
```diff
@@ -29,18 +30,20 @@ angular.module('services').service('User', [
...
-                    ) === -1,
-                };
+                user = new User(
+                  {
+                    ...result.me,
+                    firstName: result.me.firstname,
+                    lastName: result.me.name,
+                    billingCountry: result.me.country,
+                    customerCode: result.me.customerCode,
+                    isVATNeeded:
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/components/user/user.service.js]: IMO, if we have a `User` model, `isVATNeeded` could be a method of `User` as same conditions will always applies. What do you think ?

Comment (marie-j) [reply packages/manager/apps/dedicated/client/app/components/user/user.service.js]: @cbourgois I agree but I didn't spend much time on checking and refactoring the usage of `User`. Is that ok with you if we handle it in another PR ?

Comment (cbourgois) [reply packages/manager/apps/dedicated/client/app/components/user/user.service.js]: It's ok ;)

**Comment by marie-j** [comment]
Rebased and fixed conflicts

**Comment by marie-j** [comment]
I've added missing import and updated some content due to CX review

**Pull request merged**